src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -202,7 +202,7 @@ public boolean publish(String message, JSONArray attachments, String color) {
             CloseableHttpClient client = getHttpClient();
 
             try {
-                post.setEntity(new StringEntity(json.toString()));
+                post.setEntity(new StringEntity(json.toString(), "UTF-8"));
```

Comment:
can we use https://docs.oracle.com/javase/8/docs/api/index.html?java/nio/charset/StandardCharsets.html ?